BoxContentSDK/BoxContentSDKTests/BOXFileUploadNewVersionRequestTests.m

```diff
@@ -34,205 +34,205 @@ - (void)test_shared_link_properties
     XCTAssertEqualObjects([request itemTypeForSharedLink], BOXAPIItemTypeFile);    
 }
 
-- (void)test_that_upload_from_local_file_has_expected_URLRequest
-{
-    NSString *localFileName = @"tempFile.txt";
-    NSString *uploadData = @"hello";
-    
-    NSURL *temporaryDirectoryURL = [NSURL fileURLWithPath:NSTemporaryDirectory() isDirectory:YES];
-    NSURL *localFileURL = [temporaryDirectoryURL URLByAppendingPathComponent:localFileName];
-    NSError *writeError = nil;
-    [uploadData writeToURL:localFileURL atomically:YES encoding:NSUTF8StringEncoding error:&writeError];
-    XCTAssertNil(writeError);
-    
-    NSString *fileID = @"123";
-    NSString *localFilePath = [localFileURL path];
-    
-    BOXFileUploadNewVersionRequest *request = [[BOXFileUploadNewVersionRequest alloc] initWithFileID:fileID localPath:localFilePath];
-    NSURLRequest *URLRequest = request.urlRequest;
-    
-    // URL
-    NSURL *expectedURL = [NSURL URLWithString:[NSString stringWithFormat:@"%@/files/%@/content", [BOXContentClient APIUploadBaseURL], fileID]];
-    XCTAssertEqualObjects(expectedURL, URLRequest.URL);
-    XCTAssertEqualObjects(@"POST", URLRequest.HTTPMethod);
-    
-    // Multi part form body
-    XCTAssertTrue([request.operation isKindOfClass:[BOXAPIMultipartToJSONOperation class]]);
-    BOXAPIMultipartToJSONOperation *operation = (BOXAPIMultipartToJSONOperation *)request.operation;
-    
-    XCTAssertEqual(1, operation.formPieces.count);
-    
-    for (BOXAPIMultipartPiece *formPiece in operation.formPieces)
-    {
-        NSString *contentDisposition = formPiece.headers[@"Content-Disposition"];
-        NSString *bodyDataString = [self stringFromInputStream:formPiece.bodyInputStream];
-        
-        if ([contentDisposition isEqualToString:[NSString stringWithFormat:@"form-data; name=\"file\"; filename=\"\""]]) {
-            XCTAssertEqualObjects(uploadData, bodyDataString);
-        }
-        else {
-            XCTFail(@"Unexpected multipart form piece encountered: %@", formPiece);
-        }
-    }
-    
-    // HTTP Headers
-    [request.operation prepareAPIRequest]; // BOXAPIMultipartToJSONOperation does not populate headers until prepareAPIRequest
-    unsigned long long expectedContentLength = 0;
-    for (BOXAPIMultipartPiece *formPiece in operation.formPieces)
-    {
-        expectedContentLength += formPiece.contentLength;
-    }
-    NSString *expectedContentLengthString = [NSString stringWithFormat:@"%llu", expectedContentLength];
-    XCTAssertEqualObjects(expectedContentLengthString, URLRequest.allHTTPHeaderFields[@"Content-Length"]);
-    XCTAssertEqualObjects(@"multipart/form-data; boundary=0xBoXSdKMulTiPaRtFoRmBoUnDaRy", URLRequest.allHTTPHeaderFields[@"Content-Type"]);
-}
+//- (void)test_that_upload_from_local_file_has_expected_URLRequest
```

Comment:
why do we comment out those tests? The only tests that should change are relating to removed classes like BOXAPIMultipartPiece